gips/data/sentinel2/sentinel2.py

```diff
@@ -612,6 +613,11 @@ class sentinel2Data(Data):
             'bands': [{'name': band_name, 'units': Data._unitless}
                       for band_name in Asset._sensors['S2A']['indices-colors']],
         },
+        'cfmask': {
+            'description': 'Cloud mask',
```

Comment:
Might make sense to expand this to "Cloud, cloud shadow, snow, and water classification".